src/main/java/com/example/petbuddybackend/controller/CareController.java

```diff
@@ -45,7 +45,8 @@ public class CareController {
                             The result is paginated, sorted and filtered by the provided parameters.
                             
                             ## Sorting
-                            You can sort by every param in DTO excluding selectedOptions, but with some fields it will be different formatt:
+                            You can sort by every param in DTO excluding selectedOptions,
+                             but with some fields it will be different formatt:
```

Comment:
```suggestion
                            You can sort by every param in DTO excluding selectedOptions,
                            but with some fields it will be different format:
```